### PR TITLE
improve vertica content

### PR DESCRIPTION
**Pull request by Sachin-Thakur**
## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

**Comment by Sachin-Thakur** [comment]
@dataders  hi I am sorry for creating PR again and again could you please merge this PR again

File: website/docs/reference/warehouse-setups/vertica-setup.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ If you're interested in contributing, check out the source code for each reposit
 
 <h2> Installing {frontMatter.meta.pypi_package} </h2>
 
-pip is the easiest way to install the adapter: <code>pip install {frontMatter.meta.pypi_package}</code>
+pip is the easiest way to install the adapter: <code>pip install dbt-vertica</code>
```
Comment (dataders) [inline website/docs/reference/warehouse-setups/vertica-setup.md]: This seems to be rendering fine?
image

update from mirna - changing back as rendered form was showing the frontmatter code.

```suggestion
pip is the easiest way to install the adapter: <code>pip install dbt-vertica</code>
```

Comment (Sachin-Thakur) [reply website/docs/reference/warehouse-setups/vertica-setup.md]: Screenshot from 2023-01-18 00-02-07
 @dataders  doesnt seems to be on this link : https://docs.getdbt.com/reference/warehouse-setups/vertica-setup

**Review by mirnawong1 — APPROVED** [review]

File: website/docs/reference/warehouse-setups/vertica-setup.md (hunk 1 above)
Comment (mirnawong1) [inline website/docs/reference/warehouse-setups/vertica-setup.md]:
```suggestion
pip is the easiest way to install the adapter: <code>pip install dbt-vertica</code>
```

**Comment by mirnawong1** [comment]
hey @Sachin-Thakur thanks for raising this - i've merged this for you but let me know if it's not rendering correctly!

**Pull request merged**